include/mlir/IR/IntegerSet.h

```diff
@@ -96,6 +106,10 @@ class IntegerSet {
 
   MLIRContext *getContext() const;
 
+  /// Walk all of the AffineExpr's in this set's constraints. Each node in an
+  /// expression tree is visited in postorder.
+  void walkExprs(std::function<void(AffineExpr)> callback) const;
```

Comment:
use llvm::function_ref instead, constructing std::function is costly.

Reply:
Done.